src/App.tsx

```
@@ -1,27 +1,54 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 // or
 // import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goodsFromServer, setGoodsFromServer] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  function handleGetAll() {
+    getAll().then((goods: Good[]) => {
+      setGoodsFromServer(goods);
+    });
```

This code calls the API helper but doesn't handle possible errors from the promise. This violates checklist item #3: "[API] - When you make a request to the server, don't forget to handle possible errors". Add error handling (for example, `.catch(...)` on the promise or use `async/await` with `try/catch`) and update the UI/state to reflect errors so the app won't fail silently.